src/serializableStateInvariantMiddleware.test.ts

```diff
@@ -98,19 +125,12 @@ describe('serializableStateInvariantMiddleware', () => {
 
     store.dispatch(dispatchedAction)
 
-    expect(console.error).toHaveBeenCalled()
-
-    const [
-      message,
-      keyPath,
-      value,
-      action
-    ] = (console.error as jest.Mock).mock.calls[0]
-
-    expect(message).toContain('detected in an action, in the path: `%s`')
-    expect(keyPath).toBe('type')
-    expect(value).toBe(type)
-    expect(action).toBe(dispatchedAction)
+    expect(log).toMatchInlineSnapshot(`
+      "A non-serializable value was detected in an action, in the path: \`type\`. Value: Symbol(SOME_CONSTANT) 
```

Comment:
Are the backslashes actually needed here?  They should be escape characters and thus not appears in the actual output string, I would thinkg.

Reply:
It’s auto generated by Jest via `toMatchInlineSnapshot`. I could manually change that if you agree though.